roles/deploy_olm_operator/tasks/main.yml

```diff
@@ -37,6 +37,19 @@
   environment:
     KUBECONFIG: "{{ kubeconfig_path }}"
 
+- name: "Inject kube_objects on OpenShift cluster for Operator certification"
+  block:
+    - name: "Inject openshift kube_objects"
+      include_role:
+        name: inject_openshift_kube_objects
+      vars:
+        ssh_private_key: "{{ lookup('env', 'HOME') }}/.ssh/private_key"
```

Comment:
Is this file requirement described in README?

Reply:
Yes.. Please review

Reply:
maybe it will be simpler to pass var when running `ansible` instead of using `"{{ lookup('env', 'HOME') }}/.ssh/private_key`

Reply:
Sorry I donot understand which var you are talking about here

Reply:
nevermind,